### PR TITLE
Restrict from triggering export when exports are still ongoing

### PR DESCRIPTION
**Pull request by Sravanksk**

### CODE REVIEW

File: src/app/qbo/sync-export/export/export.component.ts [hunk 1]
```diff
@@ -152,17 +155,51 @@ export class ExportComponent implements OnInit {
...
+
   ngOnInit() {
     const that = this;
 
     that.isExporting = false;
     that.workspaceId = +that.route.parent.snapshot.params.workspace_id;
 
-    that.isLoading = true;
-    that.billService.getOrgDetails().subscribe((res) => {
-      that.qboCompanyName = res.CompanyName;
```
Comment (ashwin1111) [inline src/app/qbo/sync-export/export/export.component.ts:163]: why are we removing this? this is being used in the dashboard

Comment (Sravanksk) [reply src/app/qbo/sync-export/export/export.component.ts:163]: This call is not required so removed it - we already have this `qbo.component.ts`
And this is also not creating any problem with QBO Company Name which is displayed in the dashboard

Comment (ashwin1111) [reply src/app/qbo/sync-export/export/export.component.ts:163]: We have this in the exports template.
Export Expenses to {{qboCompanyName}}

Comment (Sravanksk) [reply src/app/qbo/sync-export/export/export.component.ts:163]: oh yes - missed it

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**